aws/awsutil/copy_test.go

```diff
@@ -65,16 +84,29 @@ func TestCopy(t *testing.T) {
 	assert.Equal(t, f2.A, f1.A)
 	assert.Equal(t, f2.B, f1.B)
 	assert.Equal(t, f2.C, f1.C)
+	assert.Equal(t, f2.D, f1.D)
+	assert.Equal(t, f2.E.B, f1.E.B)
+	assert.Equal(t, f2.E.D, f1.E.D)
 
 	// But pointers are not!
 	str3 := "nothello"
 	int3 := 57
 	f2.A = 100
-	f2.B[0] = &str3
```

Comment:
This, before, doesn't really test anything. Was testing that you can assign a new pointer to another pointer. Which is always not true. So this test would always pass